src/test/java/core/basesyntax/MainAppTest.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainAppTest {
-
+    public static void main(String[] args) {
+        Machine excavator = new Excavator();
+        Machine track = new Track();
+        Machine bulldozer = new Bulldozer();
+        excavator.doWork();
+        track.doWork();
+        bulldozer.doWork();
+        excavator.stopWork();
+        track.stopWork();
+        bulldozer.stopWork();
+    }
 }
```

Comment:
add an empty line at the end of the file

Reply:
done